openapi/SwarmCommon.yaml

```diff
@@ -933,6 +933,23 @@ components:
           items:
             $ref: "#/components/schemas/StatusSnapshotResponse"
 
+    StatusNeighborhoodResponse:
+      type: object
+      properties:
+        address:
+          type: string
```

Comment:
Maybe to be more specific as it is a string of a number in a binary representation?